### ISSUE TITLE
Switch from Scalariform to Scalafmt

### ISSUE THREAD

**Issue by darabos** [lynxkite/lynxkite#180]
Scalariform is not maintained. After #178 it says on every build that it reformatted 7 files, but I don't know which 7 and why they are always reformatted.

[Scalafmt](https://scalameta.org/scalafmt/) looks like a great replacement! I've tried the switch and it's very easy. It makes a lot of whitespace changes though, so I won't do it in #178.

Reading its docs made me realize that Scala 2.12 has **trailing commas!** I can't wait to use them. ❤️
